src/plots/stackedBarPlot.ts

```diff
@@ -151,7 +151,7 @@ export class StackedBar<X, Y> extends Bar<X, Y> {
       if (maximum.extent !== baselineValue) {
         // only draw sums for values not at the baseline
 
-        const text = this.labelFormatter()(maximum.extent);
+        const text = this.labelFormatter()(maximum.extent, undefined, undefined, undefined);
```

Comment:
oh, eww. This seems to not break the contract of the datumformatter and so now the formatter needs to check whether or not it is formatting an actual data value.